soci/soci_index.go

```diff
@@ -368,8 +453,9 @@ func NewIndexBuilder(contentStore content.Store, blobStore store.Store, opts ...
 // Returns the SOCI index and its metadata.
 func (b *IndexBuilder) Build(ctx context.Context, img images.Image, opts ...BuildOption) (*IndexWithMetadata, error) {
 	buildCfg := buildConfig{
-		platform: platforms.DefaultSpec(),
-		gcRoot:   true,
+		platform:     platforms.DefaultSpec(),
+		gcRoot:       true,
+		indexVersion: V1,
```

Comment:
Should this be V2 as we are moving to V2 as the default?

Reply:
No, this is the code path to build a V1 SOCI index.